modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc

```diff
@@ -0,0 +1,102 @@
+Setting filters in the following popup window is optional but helpful if you want to see only specific entries.
+You can filter on:
+
+* Name of the deprecation
+* The user that executed the query
+* Driver that was used to execute the query
+* The application name that executed the query (if set)
+* The initiation type of the query
+* The query text or parts of it
+
+Clicking on the "Fetch" button in the popup will populate the report's table with the applicable data.
+
+image::mrr-deprecation-table.png[]
+
+Each row in the table represents a query in the selected timeframe that must be changed to seamlessly migrate to the latest Aura version.
+All executions of the same query are aggregated into one row (see also the "Count" column).
+At the start of each row, clicking on the magnifying glass gives you a popup with more information about the query and suggestions on dealing with each issue.
+It also provides relevant links to the documentation for each deprecation.
+
+image::mrr-resolution-guide.png[width=600]
+
+The last column in the table of Cypher deprecations links to a view of this specific query in the Aura Query Log Analyzer tool, which can provide information on each execution of the selected query.
+The tool can only view queries on non-system databases.
+
+image::mrr-show-query-log-button.png[width=400]
+
+== Deprecated driver usage
+
+image::mrr-fetch-driver-stats.png[width=400]
+
+After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.
+By default, the filters in the popup are set to show only driver usage with potential issues in any database, including the system database.
+You can change those freely to see all driver usage, e.g.
+
+Clicking on the "Fetch" button in the popup will populate the report's table with the applicable data.
+Depending on the type of client accessing the Neo4j database, links are provided in the column “Latest version” to help with the upgrade.
+
+image::mrr-driver-table.png[]
+
+Like the Cypher deprecations table, the last column links to a view of this specific driver's executed queries in the Aura Query Log tool.
+The tool can provide information on each query execution in which the selected driver was used.
+The tool can only view queries on non-system databases.
+
+=== Deprecated index types
+
+This section provides information on how to deal with deprecated indexes that may be used in version 4 but need to be handled before or while moving to version 5.
+
+This part involves manually running a provided Cypher query on your database to identify the deprecated indexes and then deciding how to best deal with them.
+Further enhancements to this feature will be provided in the future.
+
+=== Next steps
+
+After implementing all the recommended fixes from the report, you can now test the migration.
+To do that, please follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.
```

Comment:
```suggestion
To do that, follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.
```
Which docs are you referring to?

Reply:
The docs that are linked directly from the MRR page. So far its this link: https://neo4j.com/docs/aura/tutorials/upgrade/#_perform_the_upgrade

Reply:
Alright, in that case, I suggest:
_Use the "Test migration" button at the bottom of the page and then follow the steps outlined in the docs._